x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -8,10 +8,9 @@ import { type TypeOf } from '@kbn/config-schema';
 import type { PackagePolicy, AgentPolicy } from '@kbn/fleet-plugin/common';
 import { CspFinding } from './schemas/csp_finding';
 import { SUPPORTED_CLOUDBEAT_INPUTS, SUPPORTED_POLICY_TEMPLATES } from './constants';
-import { CspRuleTemplateMetadata } from './schemas/csp_rule_template_metadata';
-import { CspRuleTemplate } from './schemas';
-import { findCspRuleTemplateRequest } from './schemas/csp_rule_template_api/get_csp_rule_template';
+
 import { getComplianceDashboardSchema } from './schemas/stats';
+import { CspRuleMetadata } from './types/latest';
```

Comment:
nit: I believe this can be `./types` as well because index returns the latest with it